test/Mono.Linker.Tests.Cases/Reflection/CoreLibMessages.cs

```diff
@@ -0,0 +1,23 @@
+using Mono.Linker.Tests.Cases.Expectations.Assertions;
+using Mono.Linker.Tests.Cases.Expectations.Metadata;
+
+namespace Mono.Linker.Tests.Cases.Reflection
+{
+	[SetupLinkerAction ("copy", "CoreLibEmulator")]
+	[SetupCompileBefore ("CoreLibEmulator.dll", new string [] { "Dependencies/CoreLibEmulator.cs" }, null, new string [] { "INCLUDE_CORELIB_IMPL" })]
+
+	// Validate that calls from one overload of Type.Get* to another overload of the same method don't produce warning
+	[LoggedMessageDidNotMatch ("Reflection call 'System.Reflection.ConstructorInfo System.Type::GetConstructor\\([^)]*\\)' inside 'System.Reflection.ConstructorInfo System.Type::GetConstructor\\([^)]*\\)' does not use detectable instance type extraction")]
```

Comment:
The name is a bit weird. I'd go with something like `LogDoesNotContain`

Reply:
Does it matter that it does regex match instead of a simple "substring"?
But I'm perfectly fine with `LogContains` and `LogDoesNotContain` names...

Reply:
It does not

Reply:
Thanks, renamed.